guides/standards-assurance-tables.md

```diff
@@ -0,0 +1,158 @@
+
+The SATs have concepts of "Objectives", "Evidence", "Confidence", an overall "Delta" (which is the most pertinent output) and "Further Evidence Required", with supporting commentary.
+
+The primary SAT purpose is to assess the system against the MCSS/CSP, determining how compliant (or non-compliant) the system is, based on a collation and review of evidence. Evidence is analysed to determine confidence that the evidence demonstrates the system meets (or does not meet) the standard, and the 'gap' (delta) between system posture according to said evidence and the standards.
+
+### Objectives
+
+The MCSS/CSPs have been distilled into 39 Objectives which the assessor (typically a cyber security professional) completing the SAT is measuring against.
+
+The [categories used within the MCSS](https://ministryofjustice.github.io/security-guidance/principles/identify-protect-detect-respond-recover/#identify-protect-detect-respond-recover) have been discussed seperately.
+
+Objectives are templated, so can be added to but not modified in-place.
+
+### Evidence
+
+To avoid assessments that are ultimately anecdotal, the SAT considers written evidence.
+
+Evidence can come in the form of transcribed conversations, diagrams, documentation or an otherwise written conveyable information about a system. 
+
+Evidence may not be related to the system itself but form a part, for example, where there is a wider document that is not system orientated that describes who relevant role holders currently are.
+
+Evidence is 'Held', 'Partial', 'Not Held' or 'N/A' (where the Objective is not applicable to the system being assessed).
+
+### Confidence
+
+The assessor reviews the evidence and uses their professional opinion to indicate a Confidence Score.
+
+The Confidence Score is based on a scale of 0 (no confidence at all) to 14 (high level of confidence), or 'N/A' (where the Objective is not applicable to the system being assessed).
+
+### Delta
+
+The Delta Rating is the resulting 'distance' between the assessed system posture against an Objective and the confidence of the same.
+
+Mathematically, the results are N/A (where the Objective is not applicable to the system being assessed) or 0 to 14 (inc).
+
+A wide delta (higher numerical value) conveys the Objective is not met. A narrow delta (lower numerical value) conveys the Objective is closer to being met.
+
+The Delta Rating is automatically calculated as '14 minus Confidence Score'.
+
+### Further Evidence Required
+
+The assessor indicates what further evidence _types_ in their view are required based on the evidence they have thus far.
+
+The [Further Evidence Required (Help) sheet](https://docs.google.com/spreadsheets/d/1oO9RAZQ6ETsN2iHZb5zcFM0HROliw18MBv89oF-BPeg/edit#gid=679100590) has a calculator which the assessor will use.
+
+This data point was designed as unique numericals as this allows programmatic analysis in the future. This data point and how it is formed is currently under active review.
```

Comment:
This data point was designed as unique numericals as this allows programmatic analysis in the future. This data point and how it is formed is currently under active review. -> The data point is currently a unique number to assist with future automated analysis. The format and range of values for the data point is currently under active review and so subject to change without notice.